contrib/core/actions/pause.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  name: "pause"
+  runner_type: "run-python"
+  description: "Action to Pause workflow"
```

Comment:
Unless i'm mistaken it just pauses the current thread of the workflow/subworkflow. 
I've had it running 10 concurrently and it doesn't do them one at a time. but i'll update the description.